docker-compose.yml

```diff
@@ -15,9 +15,8 @@ services:
       - opencopilot-net
     env_file:
       - llm-server/.env
-    ports:
-      - 8002:8002
-      - 5678:5678
```

Comment:
removing public port because we have now multiple instance and utilizing the dynamic port mapping feature by docker.

Reply:
5678 is for remote debugging i believe